ninja-core/src/main/java/ninja/AssetsController.java

```diff
@@ -127,7 +123,7 @@ private void streamOutUrlEntity(URL url, Context context, Result result) {
         // check if stream exists. if not print a notfound exception
         if (url == null) {
             context.finalizeHeadersWithoutFlashAndSessionCookie(Results.notFound());
-        } else if (url.getProtocol().equals("file") && new File(url.getPath()).isDirectory()) {
+        } else if (assetsControllerHelper.isDirectoryURL(url)) {
```

Comment:
It would be great if you could add a mockito.verify to AssetsController Test that makes sure assetsControllerHelper.isDirectoryURL has been called properly...